internal/characters/nahida/cons.go

```diff
@@ -156,7 +156,7 @@ func (c *char) c6() {
 		c.c6count++
 		ai := combat.AttackInfo{
 			ActorIndex: c.Index,
-			Abil:       "Tri-Karma Purification",
+			Abil:       "Karmic Oblivion (C6)",
 			AttackTag:  combat.AttackTagElementalArt,
 			ICDTag:     combat.ICDTagNahidaSkill,
```

Comment:
I believe Nahida C6 has its own ICD tag with default ICD group.

Reply:
Yes it has it's own tag and group, but it's just a reskin of standard ICD

Reply:
I see. Then I'd change this to standard ICD and leave a comment about how this is different in dm.